lib/ImgScreen.vue

```diff
@@ -82,6 +82,7 @@ export default {
       uiTimeout: null,
       handlers: {},
       thumbnails: false,
+      closeOnScroll: false
```

Comment:
Since previously this was the default behaviour, I would intially expect this to be set to true.